### ISSUE TITLE
Model: Toxicokinetic modelling of the transfer of non-dioxin-like polychlorinated biphenyls (ndl-PCBs) from feed and soil into laying hens and their eggs 

### ISSUE THREAD

**Issue by schuelet** [RakipInitiative/ModelRepository#540]
# Model Information

**Model Name**

Toxicokinetic modelling of the transfer of non-dioxin-like polychlorinated biphenyls (ndl-PCBs) from feed and soil into laying hens and their eggs 

**References**

[To the model!](https://knime.bfr.berlin/knime/webportal/space/Model_Curation/Download%20Uncurated%20Model?exec&pm:file_ID=120627b0-6e80-447a-a980-1d218118f501)
[Execute uncurated model!](https://knime.bfr.berlin/knime/webportal/space/Model_Curation/Execute%20Uncurated%20Model?exec&pm:file_ID=120627b0-6e80-447a-a980-1d218118f501)

**Model Uploaded By**
thomas.schueler@bfr.bund.de

# Curation Pipeline

**Model Status**
- [x] in review (publication pending)
- [x] ready for publication
- [ ] model has been uploaded and published in Zenodo KJ

## Basic Quality Control
- By Chair of Curationboard: @Racem-br

**General Check**
- [ ] Online Sanitation Check: [validate](https://knime.bfr.berlin/fskx_validator/)

**[MIRARAM](https://www.sciencedirect.com/science/article/pii/S0963996920309777?via%3Dihub) Check**
- [x] Model Name 
- [x] Model ID
- [x] Model Creator (different from author!)
- [x] Creation Date
- [x] License
- [x] Model Execution Metadata (script-language/tool/web-service API)
- [x] Reference Description (URI)
- [x] Scope (any)
- [x] Parameter Metadata (Input/Output)

**Code Of Conduct**
- [x] In case of re-implementations or re-use of digital objects without changes in the business logic of the model, the person who created the FSKX file is only listed as “Creator”. In cases where the business logic of the existing digital object was significantly modified or the Creator developed the business logic of the model himself the creator can be listed as Author as well. 
- [x] The FSKX file Creator confirmed that in case of re-implementations or re-use of digital model objects the main author of the original work was contacted and did not express objections, unless the permissions given by the license of that digital model object already provides such clearance.
- [x] An “Author” of an FSKX model is the person who created the majority of the model code or, in cases where the model code was published in an online repository, the person who is listed there as primary contact person. Other persons who contributed to the model code would be referenced as an additional “Author” in the FSKX file. In those cases where Authors contributed to the FSKX file generation (e.g. by doing quality control), they would also be listed as one of the “Creators”.

**Final Step**
- [ ] Curator assigned to this ticket

## Advanced Quality Control
**By Curator**
- Technical Curation by: @schuelet
- Scientific Curation by: @DariaSavvateeva 

**Technical Curation**
- [x] Model does not contain malicious code
- [x] Model metadata validates against FSKX schema
- [x] Model can be executed with KNIME FSK-Lab
- [x] All provided simulation scenarios can be executed
- [x] Model can be executed on RAKIP Model Repository

**Scientific Curation**
- [ ] Default simulation results are in line with reference publication
- [ ] User defined simulation settings result in reasonable results
- [x] Model annotation is sufficiently detailed

## Report
**Changes Made**

Changes made to the model in preparation for publication ([Zenodo KJ](https://zenodo.org/communities/efsa-kj))

**Additional notes**

Observations or remarks about the model or its curation

**Comment by schuelet** [comment]
@DariaSavvateeva The model has no scope (Hazard, Population group or Matrix). Please decide if that is acceptable.

**Comment by DariaSavvateeva** [comment]
> 
@schuelet 
Hazard: chemical substance
Name: ndl-PCBs
Description: Non-dioxin-like polychlorinated biphenyls (PCBs): PCB 28, PCB 52, PCB 101, PCB 138, PCB 153, and PCB 180
Matrix: soil, feed, chicken meat, chicken eggs

Scope
General info: The model was designed to simulate the kinetics of the six ndl-PCBs for young laying hens with constant laying performance before the first molt


Do I understand it correctly that I cannot modify the model metadata myself and can only execute the model to test different scenarios?

**Comment by schuelet** [comment]
@DariaSavvateeva Yes, the plan was that when changes to a model are made, the updated model is then uploaded again via [upload to curation platform](https://knime.bfr.berlin/knime/webportal/space/Model_Curation/Upload%20To%20Curation%20Platform) Workflow. 

Keep in mind though, that the new model **requires a new model id**. You can remove the model id in the FSK-Editor, it is then automatically generated, but a model id is unique across the RAKIP Model Repository.

I might create a workflow where you can edit a model online and directly have it updated.

**Comment by Racem-br** [comment]
@schuelet is the new model already uploaded?

**Comment by Racem-br** [comment]
Publisher in zenodo suggestion: Daria (BfR)

**Comment by schuelet** [comment]
Updated the model with new identifier: 66111649-23cb-4bff-acfc-2051fd96a6fa

to model:
https://knime.bfr.berlin/knime/webportal/space/Model_Curation/Download%20Uncurated%20Model?exec&pm:file_ID=66111649-23cb-4bff-acfc-2051fd96a6fa

execute model:
https://knime.bfr.berlin/knime/webportal/space/Model_Curation/Execute%20Uncurated%20Model?exec&pm:file_ID=66111649-23cb-4bff-acfc-2051fd96a6fa
